### ISSUE TITLE
Swagger-core 1.5 (Swagger Spec 2.0) support

### ISSUE THREAD

**Issue by everpeace** [gettyimages/spray-swagger#63]
I would like to hear or discuss how spray-swagger would support/migrate to swagger-core 1.5??
I understand swagger-core 1.5 did change package structures and more flexible module structures which separates each responsibilities clearly.

**Comment by mhamrah** [comment]
We don't have an immediate plans to do this, but we're happy to accept a pull request.

**Comment by rleibman** [comment]
I already put a pull request in (which also uses akka-http)

**Comment by everpeace** [comment]
Wow, looks great!   However, I understand that `swagger-core` 1.5 has already been released officially but `akka-http` hasn't.  So, I think these support will be separated.

What do you think?

**Comment by rleibman** [comment]
Yes everpeace... it hasn't been officially released, it's on RC4, which means it's really close. I'm working on an internal application that won't be released for a few months, so I thought I could afford to be on the bleeding edge a bit, I first worked on akka-http and then on swagger-core 1.5, I reailize that if I had done it in the opposite order it would have easier to incorporate swagger-core's latest. Hopefully the work I did shows you how it could be done.

**Comment by rleibman** [comment]
Spoke too soon :) Apparently akka-http was released today! At least according to this: http://doc.akka.io/docs/akka-stream-and-http-experimental/
It's still marked as experimental, but it IS out there. I just updated all my projects to it without any issues.

**Comment by pjfanning** [comment]
I have forked spray-swagger and have a new release pushed to sonatype. https://github.com/swagger-spray/swagger-spray - it uses the latest io.swagger jars.
Hopefully, this should appear in maven central over the coming days.
Thanks to the all the developers who have contributed to this project, including a few unmerged pull requests that I added in the fork.
